packages/view/backend/sirius-components-view-form-edit/src/main/resources/plugin.properties

```diff
@@ -10,248 +10,251 @@
 # Contributors:
 #     Obeo - initial API and implementation
 #################################################################################
-pluginName = Form Edit Support
-providerName = www.example.org
+pluginName=Form Edit Support
```

Comment:
No need to change all those lines

Reply:
Same remark. I took the result of the EMG generation. I should not have to process it manually afterwards.
How to have the right dev environnement?

Reply:
Honestly I have never seen anyone create an environment that works the same for everybody. I would love to have something dockerized in order to produce the same result every time but the sad truth is that everyone needs to be careful.